core/src/components/searchbar/searchbar.ionic.scss

```diff
@@ -0,0 +1,71 @@
+@use "searchbar.common";
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+
+// iOS Searchbar
+// --------------------------------------------------
+
+:host {
+  --background: #{globals.$ionic-color-neutral-100};
+  --border-radius: #{globals.$ionic-border-radius-800};
+  --box-shadow: none;
+  --cancel-button-color: #{globals.$ionic-color-neutral-800};
+  --clear-button-color: #{globals.$ionic-color-neutral-800};
```

Comment:
Yes, we don't have yet specific UI for the searchbar, these are just initial styles extrapolated from the TopBar figma examples, so that we can have some initial ground to work with on the toolbar.

I just followed the neutral-800 when in doubt for now:
<img width="1192" alt="Screenshot 2024-06-26 at 08 37 41" src="https://github.com/ionic-team/ionic-framework/assets/32780808/6db03c2a-2450-488a-bf72-cea77e2b8265">